Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/ApplePayTokenizationViewModel.swift

```diff
@@ -303,30 +345,62 @@ class ApplePayTokenizationViewModel: PaymentMethodTokenizationViewModel {
             }
         }
     }
+
+    func getShippingMethodsInfo() -> ShippingMethodsInfo {
+        guard let options = PrimerAPIConfigurationModule
+            .apiConfiguration?
+            .checkoutModules?
+            .first(where: { $0.type == "SHIPPING"})?
+            .options as? ShippingMethodOptions else {
+            return .init(shippingMethods: nil, selectedShippingMethod: nil)
+        }
+
+        // Convert to PKShippingMethods
+        let apShippingMethods = options.shippingMethods.map {
+            let method = PKShippingMethod(label: $0.name, amount: NSDecimalNumber(decimal: Decimal($0.amount/100)))
```

Comment:
[nit]: you might need to use [CurrencyLoader](https://github.com/primer-io/primer-sdk-ios/blob/master/Sources/PrimerSDK/Classes/Data%20Models/Currency/CurrencyLoader.swift) and transform amount based on minor units.